segmentstore/server/src/test/java/io/pravega/segmentstore/server/containers/StreamSegmentContainerTests.java

```diff
@@ -235,7 +235,7 @@ public void testSegmentRegularOperations() throws Exception {
                 attributeUpdates.add(new AttributeUpdate(attributeReplaceIfEquals,
                         i == 0 ? AttributeUpdateType.Replace : AttributeUpdateType.ReplaceIfEquals, i + 1, i));
                 byte[] appendData = getAppendData(segmentName, i);
-                opFutures.add(context.container.append(segmentName, appendData, attributeUpdates, TIMEOUT));
+                opFutures.add(Futures.toVoid(context.container.append(segmentName, appendData, attributeUpdates, TIMEOUT)));
```

Comment:
In this test only (not the rest), you should add some code to verify that these futures actually return the offset. All of these are executed in order and you know the length (appendData), so it should be easy to store a list of expected offsets (in order) and then compare with the results from `opFutures`.